mipengine/node/monetdb_interface/csv_importer.py

```diff
@@ -0,0 +1,301 @@
+import csv
```

Comment:
This whole module needs rewriting but let's leave it for now since it plays a secondary role.